src/main.rs

```diff
@@ -78,3 +81,6 @@ fn version() -> &'static str {
     // FIXME when we have non-nightly channels, we shouldn't hardwire the "nightly" string here.
     concat!(env!("CARGO_PKG_VERSION"), "-nightly", include_str!(concat!(env!("OUT_DIR"), "/commit-info.txt")))
 }
+fn help() -> &'static str {
+    " --version or -V to print the version and commit info\n --help or -h for this message\n No flags to start diagnostic mode\n"
```

Comment:
I would use a raw string here rather than a regular string literal so you don't need `\n`.

You could mention the command line mode which starts with any other input. I would prefer "run the RLS as a language server" to "start diagnostic mode"